torch/nn/modules/batchnorm.py

```diff
@@ -128,6 +131,8 @@ def forward(self, input: Tensor) -> Tensor:
         passed when the update should occur (i.e. in training mode when they are tracked), or when buffer stats are
         used for normalization (i.e. in eval mode when buffers are not None).
         """
+        assert isinstance(self.running_mean, torch.Tensor)
```

Comment:
This is the cause of many CI failures. Looks like `self.running_mean` can be None.

Reply:
Done! This PR requires a change in `torch/nn/functional.pyi`. Since this file is not tracked on git, I might have to open a new PR to remove this file and inline all annotations.

Reply:
You need to make them in `torch/nn/functional.pyi.in` or in `tools/pyi/gen_pyi.py` I assume? Don't understand why a new PR would be needed.

Reply:
Yeah, my mistake